src/features/simd.md

```diff
@@ -0,0 +1,179 @@
+
+As an example, here is the same function from before manually rewritten to use the SIMD intrinsics.
+
+```
+#include <wasm_simd128.h>
+
+void multiply_arrays(int* out, int* in_a, int* in_b, int size) {
+
+    for (int i = 0; i < size; i += 4) {
+
+        v128_t a = wasm_v128_load(&in_a[i]);
+
+        v128_t b = wasm_v128_load(&in_b[i]);
+
+        v128_t prod = wasm_i32x4_mul(a, b);
+
+        wasm_v128_store(&out[i], prod);
+
+    }
+
+}
+```
+
+This manually rewritten code assumes that the input and output arrays are aligned and do not alias and that size is a multiple of four. The autovectorizer cannot make these assumptions and has to generate extra code to handle the cases where they are not true, so hand-written SIMD code often ends up being smaller than autovectorized SIMD code.
+
+## Compelling Use Cases
+
+The WebAssembly SIMD proposal seeks to accelerate high compute applications like audio/video codecs, image processing applications, cryptographic applications, etc. Currently WebAssembly SIMD is experimentally supported in widely used open source projects like [Halide]([https://github.com/halide/Halide/blob/master/README_webassembly.md](https://github.com/halide/Halide/blob/master/README_webassembly.md)), [OpenCV.js]([https://docs.opencv.org/3.4/d5/d10/tutorial_js_root.html](https://docs.opencv.org/3.4/d5/d10/tutorial_js_root.html)), and [XNNPACK]([https://github.com/google/XNNPACK](https://github.com/google/XNNPACK)). [TODO: add links]
+
+Some interesting demos come from the [MediaPipe project]([https://github.com/google/mediapipe](https://github.com/google/mediapipe)) by the Google Research team.
+
+As per their description, MediaPipe is a framework for building multimodal (eg. video, audio, any time series data) applied ML pipelines. And they have a [Web version](mediapipe.page.link/web), too!
+
+One of the most visually appealing demos where it’s easy to observe the difference in performance SIMD makes, is a following hand-tracking system. Without SIMD, you can get only around 3 frames per second on a modern laptop, while with SIMD enabled you get a much smoother experience at 15-16 frames per second.
+
+![image alt text](/src/_img/simd/image_2.gif) 
+
+[https://pursuit.page.link/MediaPipeHandTrackingSimd](https://pursuit.page.link/MediaPipeHandTrackingSimd)
+
+Visit the link above in Chrome Canary with SIMD enabled to try it!
+
+Another interesting set of demos that makes use of SIMD for smooth experience, come from OpenCV - a popular computer vision library that can also be compiled to WebAssembly. They’re available by link bit.ly/opencv-camera-demos, or you can check out the pre-recorded versions below:
+
+![image alt text](/src/_img/simd/image_3.gif)
+
+Card Reading
+
+![image alt text](/src/_img/simd/image_4.gif)
+
+Invisibility cloak
+
+![image alt text](/src/_img/simd/image_5.gif)
```

Comment:
Nit: use `-` instead of underscores in file names

Reply:
Can we get these images in higher quality? GIF is probably not the best format

Reply:
Please check above where I commented on this. @dtig will share originals with me and I'll convert to MP4.

Reply:
I've added both .gif, and .mp4 formats to this PR, I've chosen to animate the gifs for now because getting the videos to render exactly how I wanted them with loop play was challenging. Suggestions on how to do this correctly are appreciated. If the gifs turn out to be sufficient, I'll delete the .mp4 versions of these files.

Reply:
`<video autoplay loop muted playsinline>` should work - that's how e.g. Giphy, Imgur, Twiter and others render "GIFs" (also MP4s in fact).